airbyte-integrations/connectors/source-slack/source_slack/source.py

```diff
@@ -265,8 +305,8 @@ def check_connection(self, logger: AirbyteLogger, config: Mapping[str, Any]) ->
 
     def streams(self, config: Mapping[str, Any]) -> List[Stream]:
         authenticator = TokenAuthenticator(config["api_token"])
-        default_start_date = pendulum.now().subtract(days=14)  # TODO make this configurable
-        threads_lookback_window = {"days": 7}  # TODO make this configurable
+        default_start_date = datetime.strptime(config["start_date"], "%Y-%m-%dT%H:%M:%SZ")
```

Comment:
why do we use datetime here as a parser, seems weird, as this is why we using pendulum

Reply:
Done